### DISCUSSION TITLE
Questions about the equations used in gas turbines.

### DISCUSSION THREAD

**Discussion by jbg5482** [oemof/tespy#513] [Q&A]
I'd like to know the mathematical equations used by compressors, combustors, and turbines. 
How do I know?

**Comment by fwitte** [comment]
Hi @jbg5482,

thanks for reaching out, overlooked this question somehow...

You can have a look into the documentation of the components. The equations applied to the model then depend on your specifications. If you fix the turbine isentropic efficiency, then that equation will be applied, if you do not specify a value for it, then the efficiency will be a result of the calculation.

Best

Francesco
